### ISSUE TITLE
Mobile navigation hamburger menu. 

### ISSUE THREAD

**Issue by ChrisMunozCodes** [ChrisMunozCodes/PixelVisionize#61]
Convert to a hamburger menu navigation style on smaller screen sizes.

**Comment by uday-kumar07** [comment]
hey hi can you assign this issue to me 😊

**Comment by ChrisMunozCodes** [comment]
> hey hi can you assign this issue to me 😊

For sure! Feel free to reach out if you have any questions. I assigned you to issue #36.

**Comment by uday-kumar07** [comment]
thank you 😍💓
can you give me 1 days to complete this issue

**Comment by uday987** [comment]
hi Chris Munoz there is an issue when hovering over generate button its scales up and shows the scrollbar

can I remove the scale property?

**Comment by ChrisMunozCodes** [comment]
> i
Yes, thank you! That would be much appreciated! Good work spotting that! 😄

**Comment by ChrisMunozCodes** [comment]
> hi Chris Munoz there is an issue when hovering over generate button its scales up and shows the scrollbar
> 
> can I remove the scale property?

Hey uday987! In the latest updated repo on the development branch the scale issue was fixed and there was some css updates as well, make sure to pull the changes from the development branch.
